bayesian/utils.py

```diff
@@ -49,6 +47,9 @@
     "Do you want to consider adding this Package?"
 
 zero_version = sv.Version("0.0.0")
+AUTH_KEY = os.getenv('OS_AUTH_KEY', '')
```

Comment:
@yzainee is AUTH_KEY used in the code?

Reply:
not currently. Because the ingestion API dont have any security check at present. Once we add that, we would need some sort of authentication to be sent.. or we can remove the route from the ingestion API and make it an internal API call.